swerve100/src/main/java/org/team100/frc2023/subsystems/SwerveModuleFactory.java

```diff
@@ -54,8 +54,8 @@ public SwerveModule WCPModule(
 
         // TURNING PID
         ProfiledPIDController turningController = new ProfiledPIDController(
-                1, // kP: low P because not much reduction gearing.
-                1, // kI
+                210, // kP: High P to keep the measurments acurate while maintaining an agresive wheel turning
```

Comment:
maybe link to a doc where you show some data?  i still don't see how this isn't saturating the controller.  what's the maximum possible speed?  what's the max error you've seen?